tasks/jasmine-chromeapp.js

```diff
@@ -7,73 +7,169 @@ module.exports = function (grunt) {
     chalk = require('chalk'),
     path = require('path'),
     async = require('async'),
-    fs = require('fs-extra'),
+    fs = require('fs'),
     pkg = require('../package.json'),
-    chrome = require('node-chrome-runner');
+    chrome = require('node-chrome-runner'),
+    fileSyncCmp = require('file-sync-cmp');
 
-  function addFiles(files, to, tagFilter) {
-    var tags = '';
+  /* copyFiles is adapted from:
```

Comment:
This seems like it might make sense in a separate file. it's a rather involved method that could be used more generally.